src/types/Env.ts

```diff
@@ -13,11 +15,14 @@ export type Env = {
   DATABASE_USERNAME: string;
   TOKEN_SERVICE: Fetcher;
   AUTH_URL: string;
+  // Can we remove this? only used by src/routes/login - what is this?
   AUTH_TEMPLATES: R2Bucket;
+  // what is this?
```

Comment:
```suggestion
  AUTH_TEMPLATES: R2Bucket;
```

:eyes: both these seem used

Reply:
Yepp, if we move the email and universal auth to be bundled we should be able to drop this.